Content.Goobstation.Common/MartialArts/MartialArtsComponents.cs

```diff
@@ -50,6 +50,10 @@ public sealed partial class MartialArtsKnowledgeComponent : GrabStagesOverrideCo
     [AutoNetworkedField]
     public string OriginalFistDamageType;
 
+    //omu start
+    [DataField]
+    public bool Removable = true;
```

Comment:
If your intention is to make this change only work on the equippable CQC items, like the one that @Cygnye added, then you should work around the ones that already exist. It shouldn't be innately removable unless there is a component making it so. You have this being set to whatever the component is, but it shouldn't be true unless that component exists, so it should be defaulted to false.
```suggestion
    [DataField]
    public bool Removable = false;
```